package.json

```diff
@@ -39,6 +39,7 @@
   },
   "devDependencies": {
     "eslint": "~7.32.0",
+		"jsdom-testing-mocks": "^1.11.0",
```

Comment:
Use a tilde (~) here instead of a caret (^) so we can maintain the package version. 

This dependency should live in ncids-js instead.

Reply:
Oh and please correct this commit title and Closes # to point to the correct ticket.